apps/tailwind-components/components/table/TableEMX2.vue

```diff
@@ -64,67 +64,61 @@ function handlePagingRequest(page: number) {
-        <tr>
-          <th
-            v-for="column in columns"
-            class="py-2.5 px-2.5 border-b border-gray-200 first:pl-0 last:pr-0 sm:first:pl-2.5 sm:last:pr-2.5 text-left w-64"
-            :ariaSort="
-              settings.orderby.column === column.id
-                ? mgAriaSortMappings[settings.orderby.direction]
-                : 'none'
-            "
-            scope="col"
-          >
-            <button
-              class="overflow-ellipsis whitespace-nowrap overflow-hidden hover:cursor-pointer"
-              @click="handleSortRequest(column.id)"
+
+  <div class="overflow-auto rounded-b-50px">
+    <div class="overflow-x-auto overscroll-x-contain bg-table rounded-t-3px">
+      <table class="text-left table-fixed w-full">
+        <thead>
+          <tr>
+            <th
+              v-for="column in columns"
+              class="py-2.5 px-2.5 border-b border-gray-200 first:pl-0 last:pr-0 sm:first:pl-2.5 sm:last:pr-2.5 text-left w-64"
+              :ariaSort="
+                settings.orderby.column === column.id
+                  ? mgAriaSortMappings[settings.orderby.direction]
+                  : 'none'
+              "
+              scope="col"
             >
-              {{ column.label }}
-              <ArrowUp
-                v-if="
-                  column.id === settings.orderby.column &&
-                  settings.orderby.direction === 'ASC'
-                "
-                class="w-4 h-4 inline-block"
-              />
-              <ArrowDown
-                v-if="
-                  column.id === settings.orderby.column &&
-                  settings.orderby.direction === 'DESC'
-                "
-                class="w-4 h-4 inline-block"
-              />
-            </button>
-          </th>
-        </tr>
-      </thead>
-      <tbody>
-        <tr v-for="row in rows">
-          <TableCellTypesEMX2
-            v-for="column in columns"
-            :scope="column.key === 1 ? 'row' : null"
-            :metaData="column"
-            :data="row[column.id]"
-          />
-        </tr>
-      </tbody>
-    </table>
+              <button
+                class="overflow-ellipsis whitespace-nowrap overflow-hidden hover:cursor-pointer"
+                @click="handleSortRequest(column.id)"
+              >
+                {{ column.label }}
+                <ArrowUp
+                  v-if="
+                    column.id === settings.orderby.column &&
+                    settings.orderby.direction === 'ASC'
+                  "
+                  class="w-4 h-4 inline-block"
+                />
+                <ArrowDown
+                  v-if="
+                    column.id === settings.orderby.column &&
+                    settings.orderby.direction === 'DESC'
+                  "
+                  class="w-4 h-4 inline-block"
+                />
+              </button>
+            </th>
+          </tr>
+        </thead>
+        <tbody class="mb-3">
```

Comment:
To remove the border from the last row, you can structure the class like this -

```
<tbody class="mb-3 [&_tr:last-child_td]:border-none">
```

I found that the text feels compressed. We could also add a little bit of space on the bottom: 

```
<tbody class="mb-3 [&_tr:last-child_td]:border-none [&_tr:last-child_td]:mb-5">
```